docs/whats-new.md

```diff
@@ -41,17 +41,21 @@ luma.gl can now load 3D models and scenegraphs in the popular [glTF™](https://
 * LAS/LAZ
 * OBJ
 
-These loaders, while technically independent of luma.gl, return parsed data in a format that can be passed directly to luma.gl classes like `Geometry` and `Model`.
+loaders.gl output can now be passed directly into luma.gl classes like `Geometry` and `Model` making it straightforward to use luma.gl with a wide variety of file formats.
 
 
-### "Asynchronous" Textures
+### Asynchronous Textures
 
-The image data needed by the `Texture` classes can now be supplied using URLs (`string` or `Promise`). This can removes the need for applications to use callbacks etc to load textures.
+Image data for `Texture` classes can now be supplied using URLs or `Promise`s, making unnecessary for applications to handle loading themselves.
```

Comment:
- missing word `it`: making it unn... 
- missing word `image`: handle image loading...